python/cudf/cudf/_libxx/cpp/types.pxd

```diff
@@ -20,6 +20,10 @@ cdef extern from "cudf/types.hpp" namespace "cudf" nogil:
         ASCENDING "cudf::order::ASCENDING"
         DESCENDING "cudf::order::DESCENDING"
 
+    ctypedef enum include_nulls "cudf::include_nulls":
+        EXCLUDE_NULLS "cudf::include_nulls::NO"
+        INCLUDE_NULLS "cudf::include_nulls::YES"
```

Comment:
Why are we renaming these?

Reply:
because there is already another typedef with YES and NO in same file.

Reply:
Yea Cython complains, but we've verified it still generates proper C++ code without issue so I'd suggest we don't rename them. We're going to explore what it would take to add enum class support to Cython upstream.